test/fixtures/rest/app/apis/users/posts/answer/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+exports.index = function* () {
```

Comment:
The testcase hasn't been modified, I think the filepath should be `apis/users/posts/replies/answer.js`. I don't know why `answer` is sibling of `replies`, not the child of `replies`.